src/components/molecules/NewsFlashFilterPanel.tsx

```diff
@@ -21,6 +21,15 @@ const useStyles = makeStyles({
     width: '50px',
     border: `1px solid ${borderColor}`,
   },
+
+  buttonClicked: {
+    height: '50px',
```

Comment:
why setting size for `buttonClicked`? size should be the same.
maybe you're missing `box-sizing: border-box`?